sources/packages/backend/apps/api/src/route-controllers/application/application.students.controller.ts

```diff
@@ -555,6 +569,8 @@ export class ApplicationStudentsController extends BaseController {
       exceptionStatus: application.applicationException?.exceptionStatus,
       appealStatus,
       scholasticStandingChangeType: scholasticStandingChange?.changeType,
+      applicationOfferingChangeRequestStatus:
+        mostRecentApplicationOfferingChangeRequestStatus,
```

Comment:
IMO, I don't think you need to specify `most recent`. Because, status are always most recent

Reply:
I agree with @ann-aot. We can follow the same from appeals.